### ISSUE TITLE
i cannot click on friendly warlock summon. shows as clickable bu...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#8257]
> i cannot click on friendly warlock summon. shows as clickable but nothing happens

.go xyz -13130.400000 -611.032000 9.578000 0 2.750720

**Comment by Bennylavaa** [comment]
Warlock and Summoning Stones basically work the same on Epoch so this might be related as well: https://github.com/Project-Epoch/bugtracker/issues/7539

However warlock summoning on Epoch has a few different issues:

- Clickers do not show and animation when clicking on the portal
- After the portal has recieved the correct number of clicks the warlock (or summoner) will then get a casting bar for the summon and can move around during this however they must keep their summon target until the cast has finished. This is not how it should work in a vanilla setting. 

Basically clickers should show an animation when clicking the portal until the summon has finished. Once the summon finishes the summon should be sent out without a extra cast bar timer.

**Comment by Shadownflame** [comment]
Interesting, I had no issue summoning people on the beta. The animations were off indeed, but people not being able to click at all must have been an isolated issue.

**Comment by Shadownflame** [comment]
UPDATE: not an isolated issue, but only affects warlock summon and not meeting stones. Has something to do with PVP flags and cross faction most likely, although the test results are very inconsistent. Depending on who has flag and who hasn't, and who is being targeted, always different results. It's unclear whether the warlock summon currently needs 1 or 2 persons to click as well. The first clicker usually doesn't get an animation and the 2nd person can either click and get the spell off or not, but sometimes the first click seems enough to get the spell off.

**Comment by Shadownflame** [comment]
No changes here.

**Comment by Imperius5k** [comment]
confirmed

**Comment by Shadownflame** [comment]
Didn't encounter any issues with this anymore. If someone does, please reopen.
